Spanner/tests/System/PgPartitionedDmlTest.php

```diff
@@ -30,6 +30,11 @@ class PgPartitionedDmlTest extends SpannerPgTestCase
 
     public function testPdml()
     {
+        // Skipping temporarily while we figure out the issue with
+        // `executePartitionedUpdate` not accepting parameters in
+        // PG format.
+        self::skipEmulatorTests();
```

Comment:
nit:  use instance method for consistency. 
```suggestion
        $this->skipEmulatorTests();
```